.gitignore

```diff
@@ -1,5 +1,7 @@
+.puli/
 build/
 vendor/
 composer.lock
 phpspec.yml
 phpunit.xml
+puli.json
```

Comment:
Ok, makes sense then.

The package list is in there since, at least in principle, Puli is independent of Composer. You could install additional packages manually or generally use Puli without Composer, if that makes sense for your project. So puli.json duplicates some parts of composer.json, however those parts are synchronized by the Composer plugin.